poutyne/framework/model.py

```diff
@@ -187,7 +195,7 @@ def fit(self,
             callbacks=None):
         # pylint: disable=line-too-long
         """
-        Trains the model on a dataset. This method creates generators and calls
+        Trains the network on a dataset. This method creates generators and calls
```

Comment:
This is inconsistent with the doc of `fit_generator`.

Reply:
solved in 3e0ae6f